deploy/helm/sumologic/conf/logs/logs.source.containers.conf

```diff
@@ -48,7 +48,19 @@
     exclude_container_regex "#{ENV['EXCLUDE_CONTAINER_REGEX']}"
     exclude_host_regex "#{ENV['EXCLUDE_HOST_REGEX']}"
   </filter>
-  <match **>
+  # <filter containers.**>
+  #   @type record_modifier
+  #   <record>
+  #     message ${record["message"]["log"]}
+  #   </record>
+  # </filter>
+  # <filter containers.**>
+  #   @type record_modifier
+  #   <record>
+  #     message ${begin; record["message"].merge(JSON.parse(record["message"]["log"])).delete("log"); rescue JSON::ParserError; record["message"]; end}
+  #   </record>
+  # </filter>
+  <match containers.**>
```

Comment:
did we want to add before and after examples for each filter here?

Reply:
good catch, will do that